src/Tizen.Multimedia.Util/ImageUtil/ImageDecoder.cs

```diff
@@ -448,4 +448,26 @@ public HeifDecoder() : base(ImageFormat.Heif)
 
         internal override int HeaderOffset => 4;
     }
+
+    /// <summary>
+    /// Provides the ability to decode the JPEG (Joint Photographic Experts Group) XL encoded images.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    public class JpegXlDecoder : ImageDecoder
+    {
+        private static readonly byte[] _header = { (byte)'J', (byte)'X', (byte)'L'};
```

Comment:
The JPEG XL file format can take two forms:
The ISO-based jpeg xl image will start with the 'jxl'.
Another jpeg xl images called 'naked' codestream start with 0xFF0A.
Please refer to below native code.

- image_util_decode.c
static unsigned char _JXL_CODESTREAM_HEADER[] = { 0xFF, 0x0A };
{ IMAGE_UTIL_JPEG_XL, _JXL_CODESTREAM_HEADER, sizeof(_JXL_CODESTREAM_HEADER), 0 },